code/modules/surgery/organs/tongue.dm

```diff
@@ -92,6 +92,54 @@
 		message = lizard_eckS.Replace(message, "ECKS$1")
 	speech_args[SPEECH_MESSAGE] = message
 
+/obj/item/organ/tongue/lizard/silver
+	name = "silver tongue"
+	desc = "A genetic branch of the high society Silver Scales that gives them their silverizing properties. To them, it is everything, and society traitors have their tongue forcibly revoked. Oddly enough, it itself is just blue."
+	icon_state = "silvertongue"
+	actions_types = list(/datum/action/item_action/organ_action/statue)
+
+/datum/action/item_action/organ_action/statue
+	name = "Become Statue"
+	desc = "Become an elegant silver statue. Its durability and yours are directly tied together, so make sure you're careful."
```

Comment:
... are the statue's durability and your durability actually tied together? is that handled automatically by petrification code? what happens if you transform back while the statue is merely damaged, not destroyed?